### PR TITLE
Branch three

### PR DESCRIPTION
**Pull request by BereketRetta**
I built a simple HTML list of To Do tasks. This simple web page was built using webpack and served by a webpack dev server.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -31,21 +23,82 @@ class Todo {
...
+  } else if (e.target.classList.contains('items') && !(e.target.classList.contains('delete'))) {
+    let id = e.target.parentElement;
+    id = id.previousElementSibling;
+    id = id.id;
+    Store.makeEditable(e.target, id);
+  }
+});
+
+function add(e) {
+  const task = document.querySelector('#myInput').value;
```
Comment (Mihndim2020) [inline src/index.js:84]: - [ ] As it is a requirement for this project, and a best practice to make your code more modular, please kindly create a new JavaScript file for the new functionality(`adding a task`, `editing a task` and `deleting a task`). See screenshot below and kindly fix that.
todoFile

File: src/index.js [hunk 2]
```diff
@@ -31,21 +23,82 @@ class Todo {
...
-  static addToDo(item) {
-    listContainer.innerHTML += `<div class="list-container">
-    <span>
-    <input type="checkbox">
-    <p class="items">${item.todoText}</p>
-    </span>
-    <p id="${item.index}"></p>
-    <i class="fas fa-ellipsis-v"></i>
-    </div>`;
+  static deleteTask(id) {
```
Comment (Mihndim2020) [inline src/index.js:56]: - [ ] As it is a requirement for this project, and a best practice to make your code more modular, please kindly create a new JavaScript file for the new functionality(`adding a task`, `editing a task` and `deleting a task`). See screenshot below and kindly fix that.
todoFile

File: src/index.js [hunk 3]
```diff
@@ -31,21 +23,82 @@ class Todo {
...
+  static displayTask() {
+    Todo.sortTasks();
+    const todos = Store.getTasks();
+    listContainer.innerHTML = '';
+    todos.forEach((item) => Todo.addTask(item));
+  }
 
-    const todos = toDoObjectList;
-    todos.forEach((item) => Todo.addToDo(item));
+  static addTask(item) {
```
Comment (Mihndim2020) [inline src/index.js:36]: - [ ] As it is a requirement for this project, and a best practice to make your code more modular, please kindly create a new JavaScript file for the new functionality(`adding a task`, `editing a task` and `deleting a task`). See screenshot below and kindly fix that.
todoFile

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Your app is running and free of `Linter` errors.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Please in your next milestones, kindly give descriptive branch names to your branches as this will enable others working on the project to have an idea of what the branch is about. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BereketRetta** [comment]
Hello, @Mihndim2020 Thank you for your review, But I wanted to tell you that I implemented the modular code structure in the addremove.js file. can you kindly check that out?

**Comment by BereketRetta** [comment]
Screen Shot 2022-01-19 at 2 03 39 PM

**Review by mikethreels — APPROVED** [review]
Hi Bereket, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**